### DISCUSSION TITLE
Possible move of django-helpdesk to Django Commons

### DISCUSSION THREAD

**Discussion by uhurusurfa** [django-helpdesk/django-helpdesk#1216] [Announcements]
I would like to get opinions on moving this project into the Django Commons fold:
https://github.com/django-commons

Does anyone have any strong opinions on this?

**Comment by Benbb96** [comment]
As for me, I don't see why it would be a bad idea. It looks like transferring everything would be a bit of work, making sure not to break anything in the process. But in the end, integration into a larger community could help maintain the project. 😃

There's only this other repo in the organization : https://github.com/django-helpdesk/django-bootstrap5-form that I think is not used by django-helpdesk :
https://github.com/django-helpdesk/django-helpdesk/blob/b02e8d416275b7006bbd584f9a2643ac8e051377/requirements.txt#L2

So maybe the django-helpdesk org would disappear?

**Reply by uhurusurfa** [reply]
Since the Django Commons is still within Github, I doubt there will be much to do to move it over.
There is Jazzband  (https://github.com/jazzband/) attempting to do the same thing but my experience with them is that there is a single person running it and he seems to be a control freak that has the consequence of severely limiting contributions and upkeep of the apps that have moved to Jazzband and they have some very strange ideas about what is required for someone to manage a project in their "community" resulting in many projects no longer being maintained at all.

**Comment by tim-schilling** [comment]
Hi folks! I found this discussion and wanted to see if you had any questions about how Django Commons works or any concerns with what you would expect?

**Reply by uhurusurfa** [reply]
Seems clear enough to me. I will reach out to you for advice when I have some time to dedicate towards making the move.
